src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -156,11 +171,12 @@ export default class HexagonLayer extends Layer {
   }
 
   updateUniforms() {
-    const {opacity, enable3d, invisibleColor} = this.props;
+    const {opacity, extruded, invisibleColor, radiusScale} = this.props;
     this.setUniforms({
-      enable3d: enable3d ? 1 : 0,
+      extruded: extruded ? 1 : 0,
```

Comment:
Just curious, do we need to do this with boolean uniforms? Doesn't luma handle that?